<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LecteurVue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <item>
     <widget class="QLabel" name="lTitreDiapo">
      <property name="text">
       <string>titreDiapo</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,1,0,2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lTitreImage">
        <property name="text">
         <string>titre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lCategorieImage">
        <property name="text">
         <string>Catégorie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lRangImage">
        <property name="text">
         <string>Rang</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lTotalNbImages">
        <property name="text">
         <string>/TotalnbImages</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="text">
         <string>image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0">
      <item>
       <widget class="QLabel" name="lMode">
        <property name="text">
         <string>Mode: Manuel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bLancerDiapo">
        <property name="text">
         <string>Lancer diaporama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bArreterDiapo">
        <property name="text">
         <string>Arreter Diaporama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPrecedent">
        <property name="text">
         <string>Précédent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSuivant">
        <property name="text">
         <string>Suivant</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="acQuitter"/>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Paramètres</string>
    </property>
    <addaction name="acCharger"/>
    <addaction name="acVider"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="acAproposde"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="acQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="acCharger">
   <property name="text">
    <string>Charger diaporama</string>
   </property>
  </action>
  <action name="acVider">
   <property name="text">
    <string>Enlever diaporama</string>
   </property>
  </action>
  <action name="acAproposde">
   <property name="text">
    <string>A propos de ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acQuitter</sender>
   <signal>triggered()</signal>
   <receiver>LecteurVue</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
